packages/pds/src/account-manager/helpers/account.ts

```diff
@@ -64,6 +64,41 @@ export const getAccount = async (
   return found || null
 }
 
+export const getAccounts = async (
+  db: AccountDb,
+  handlesAndDids: string[],
+  flags?: AvailabilityFlags,
+): Promise<Map<string, ActorAccount>> => {
+  const results = new Map<string, ActorAccount>()
+  const handles = new Set<string>()
+  const dids = new Set<string>()
+  for (const handleOrDid of handlesAndDids) {
+    if (handleOrDid.startsWith('did:')) {
+      dids.add(handleOrDid)
+    } else {
+      handles.add(handleOrDid)
+    }
+  }
+
+  const accounts = await selectAccountQB(db, flags)
+    .if(handles.size > 0, (qb) =>
+      qb.where('actor.handle', 'in', Array.from(handles)),
```

Comment:
I'm pretty sure these two `.where`s strung together are going to turn into a `WHERE ... AND ...`

Kysely has an `orWhere` function, but sometimes the grouping around those can be a little bit tricky. Partially why I recommend just ditching the handles 😛